Modules/Filtering/Thresholding/include/itkThresholdImageFilter.h

```diff
@@ -103,15 +103,15 @@ class ITK_TEMPLATE_EXPORT ThresholdImageFilter : public InPlaceImageFilter<TImag
   /** Get the "outside" pixel value. */
   itkGetConstMacro(OutsideValue, PixelType);
 
-  /** The values greater than or equal to the value are set to OutsideValue. */
+  /** The values greater than "thresh" are set to OutsideValue. */
   void
   ThresholdAbove(const PixelType & thresh);
```

Comment:
```suggestion
  ThresholdAbove(const PixelType & threshold);
```

Reply:
This sort of change only affects the documentation. I do remember we have a maintenance script which synchronizes the name in the interface and implementation, but it might be copying the implementation name to the interface.

Reply:
I think those keyword arguments are named after the method names, not method arguments' names.